.github/CODEOWNERS

```diff
@@ -25,7 +25,7 @@
 #/avm/res/compute/disk/ @Azure/avm-res-compute-disk-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/compute/disk-encryption-set/ @Azure/avm-res-compute-diskencryptionset-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/compute/gallery/ @Azure/avm-res-compute-gallery-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/compute/image/ @Azure/avm-res-compute-image-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/compute/image/ @Azure/avm-res-compute-image-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Please ref to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
@Azure/avm-res-compute-image-module-owners-bicep
@Azure/avm-res-compute-image-module-contributors-bicep